earth_enterprise/src/common/khxml/khxml.h

```diff
@@ -23,6 +23,8 @@
 #include <qstring.h>
 
 namespace khxml = xercesc;
+// forward declaration
```

Comment:
I think all the changes in this file can be reverted. It looks like they all have to do with the experimental method of reinitializing xerces, which we're not keeping in the code right now.